### PR TITLE
lift-analytics-Maps using Json files

### PR DESCRIPTION
**Pull request by Soumyadeep-Oracle**

### CODE REVIEW

File: data-platform/analytics/oac-map-extensions/fles/Geo Json Files.zip
Comment (AlexanderHodicke) [inline data-platform/analytics/oac-map-extensions/fles/Geo Json Files.zip:1]: @oheimburger can you have a look at the geo JSON stuff

**Review by AlexanderHodicke — CHANGES_REQUESTED** [review]

**Comment by oheimburger** [comment]
Whare are the GeoJSON files from? Is there a Copyright?

**Comment by oheimburger** [comment]
The files in the directory `fles` should be moved to `files`. Directory `fles` should be removed.

**Comment by Soumyadeep-Oracle** [comment]
> Whare are the GeoJSON files from? Is there a Copyright?

These geo jsons have been taken as a sample from d3 and modified according to our requirements for different countries.
D3 is a free tool available.

**Review by oheimburger — CHANGES_REQUESTED** [review]

**Comment by Soumyadeep-Oracle** [comment]
Changes done.

From: oheimburger ***@***.***>
Date: Tuesday, 18 July 2023 at 6:33 PM
To: oracle-devrel/technology-engineering ***@***.***>
Cc: Soumyadeep Mitra ***@***.***>, Author ***@***.***>
Subject: [External] : Re: [oracle-devrel/technology-engineering] lift-analytics-Maps using Json files (PR #86)

@oheimburger requested changes on this pull request.

________________________________

On data-platform/analytics/oac-map-extensions/THIRD_PART_LICENSES<https://urldefense.com/v3/__https:/github.com/oracle-devrel/technology-engineering/pull/86*discussion_r1266742454__;Iw!!ACWV5N9M2RV99hQ!LEyW4IPyCTGIMTo2wa2ChJkPQRjVc1ffNBP43gN91gzAMt8-Zi1oLCIn2VMxkqlif6bMLi-A_-aj54QtGu-eblUdyMMnCA$>:

Change this file name to 'THIRD_PARTY_LICENSES'.

**Review by oheimburger — DISMISSED** [review]

**Review by AlexanderHodicke — APPROVED** [review]

**Pull request merged**